frontent/src/Website/CommentBox.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import './MovieDetails.css';
+import logoimage from "../aessest/UserPNG.png";
+
+const CommentBox = () => {
+  const [commentValue, setCommentValue] = useState('');
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    const fetchComments = async () => {
```

Comment:
why fetchComments function defined inside of useEffect cant we define fetchComments outside and call the function inside useEffect?